seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/FileUtils.java

```diff
@@ -50,6 +54,19 @@ public static Path getConfigPath(AbstractCommandArgs args) {
         }
     }
 
+    /**
+     * Check whether the conf file exists.
+     *
+     * @param configFile the path of the config file
+     */
+    public static void checkConfigExist(Path configFile) {
+        if (!configFile.toFile().exists()) {
+            String message = "Can't find config file: " + configFile;
+            log.error(message);
+            throw new SeaTunnelException(message);
```

Comment:
`SeaTunnelRuntimeException(CommonErrorCode.FILE_OPERATION_FAILED, message)` is better.

Reply:
Thanks for you suggestion.I have fixed it.